### ISSUE TITLE
Error evaluating function `unit`: the first argument to unit must be a number

### ISSUE THREAD

**Issue by ParthS007** [DocSpring/craco-less#73]
Hi 👋 , we use `semantic-ui-react` in our application and one of the dependency is `@semantic-ui-react/craco-less` and I am getting this error after the latest `1.18.0` release.

```
./node_modules/semantic-ui-less/semantic.less (./node_modules/css-loader/dist/cjs.js??ref--5-oneOf-9-1!./node_modules/postcss-loader/src??postcss!./node_modules/less-loader/dist/cjs.js??ref--5-oneOf-9-3!./node_modules/semantic-ui-less/semantic.less)

@relativeLarge   : unit( round(@largeSize * @emSize) / @emSize, em);
@relativeBig     : unit( round(@bigSize * @emSize) / @emSize, em);
                 ^
Error evaluating function `unit`: the first argument to unit must be a number. Have you forgotten parenthesis?
      Error in .. /node_modules/semantic-ui-less/themes/default/globals/site.variables (line 408, column 19)
```

This error got solved by pinning the version `1.17.1` like this, thanks to @mvidalgarcia

```
...
"resolutions": {
    "@semantic-ui-react/craco-less/craco-less": "1.17.1"
   }
...
```

I am not sure what's the root cause of the error, maybe something wrong on semantic side(I posted similar https://github.com/Semantic-Org/Semantic-UI-LESS/issues/74 there). Just posting here in case if it's a bug or new requirement of `unit` function.

**Comment by gdelacruzfdez** [comment]
Same issue here, also using ```@semantic-ui-react/craco-less```.

**Comment by mvidalgarcia** [comment]
Should be solved by https://github.com/layershifter/semantic-ui-tools/pull/16 already published in `@semantic-ui-react/craco-less@1.2.3`
